client/src/main/kotlin/io/spine/examples/chatspn/App.kt

```diff
@@ -37,25 +23,38 @@ import androidx.compose.ui.unit.dp
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+package io.spine.examples.chatspn
+
+import androidx.compose.desktop.ui.tooling.preview.Preview
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import io.spine.examples.chatspn.chat.ChatsPage
 
+val chatProvider = TestDataChatProvider()
+val userProvider = TestDataUserProvider()
+
+/**
+ * The root component of the application.
+ *
+ * Responsible for navigation and composition of pages.
+ */
 @Composable
 @Preview
 fun App() {
-    val count = remember { mutableStateOf(0) }
-    MaterialTheme {
-        Column(Modifier.fillMaxSize(), Arrangement.spacedBy(5.dp)) {
-            Button(modifier = Modifier.align(Alignment.Start),
-                onClick = {
-                    count.value++
-                }) {
-                Text(if (count.value == 0) "Hello World" else "Clicked ${count.value}!")
-            }
-            Button(modifier = Modifier.align(Alignment.Start),
-                onClick = {
-                    count.value = 0
-                }) {
-                Text("Reset")
-            }
-        }
+    var page by remember { mutableStateOf(Page.CHATS) }
+    when (page) {
+        Page.CHATS -> ChatsPage(userProvider, chatProvider)
+    }
+    // Startups the testing chatting emulation
+    LaunchedEffect(Unit) {
+        chatProvider.startChatting()
     }
 }
+
+enum class Page {
```

Comment:
Starting the next PR, let's use Kotlin's explicit API mode. The Kotlin compiler should also be configured with this setting enabled via Gradle.

@Artem-Semenov-dev FYI.